libs/langgraph-checkpoint-mongodb/tests/test_pregel.py

```diff
@@ -0,0 +1,12030 @@
+import enum
```

Comment:
could you remove this and `test_pregel_async`? these will quickly drift from the library anyway. we'll introduce a library with a standard test suite soon and you should be able to make use of that

Reply:
I would prefer to keep these for now. I put a lot of effort into them and they provide code test coverage that the unit tests do not. I'm happy to refactor this later when the standard test suite is ready.